metadata.json

```diff
@@ -14,6 +14,13 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
```

Comment:
Will this work for RHEL 6.5, 6.6. etc? Perhaps a better fact to use is "operatingsystemmajrelease"